src/main/java/com/service/CreatePersonService.java

```diff
@@ -154,4 +187,72 @@ public EStatus createPersonStatus() {
             }
         }
     }
+
+    public void readPropertyFileAndExecuteReadFileLinesMethod() throws IOException {
+        String testFilePath = CreatePropertiesFile.getInstance().getProperty("testFilePath");
+        BufferedReader bufferedReader = new BufferedReader(new FileReader(testFilePath));
+        readFileLines(bufferedReader);
+
+    }
+
+    public void readFileLines(BufferedReader bufferedReader) throws IOException {
```

Comment:
Does this have to be public? Also same story for the exceptions. When you are throwing exceptions you should think about the guy who is going to handle them